### ISSUE TITLE
 'Assignment to read-only properties is not allowed in strict mode' error in ie11 private mode 

### ISSUE THREAD

**Issue by sharathic** [ujjwalguptaofficial/JsStore#112]
'Assignment to read-only properties is not allowed in strict mode' error comes in the console when  opened in IE11 private mode,, Looking forward for the fix

thank you

**Comment by ujjwalguptaofficial** [comment]
Are you using jsstore latest version ? please confirm

**Comment by ujjwalguptaofficial** [comment]
Fixed in v - 3.1.3. 

Ie does not support indexeddb in private mode. So jsstore will detect this & throw a error in `initDb` api. You need to change your app code based on indexeddb support.

e.g - 

```
var con = new JsStore.Instance();
var isIndexedDbSupported = true;
try {
    await con.initDb(db_schema);
} catch (ex) {
    isIndexedDbSupported = false;
    alert('Browser does not support indexeddb');
}
```

Note : this applies for all browser who does not support indexeddb.
